R/derive_var_ontrtfl.R

```diff
@@ -1,81 +1,91 @@
-#' Optional; default is `NULL`.
-#'
-#' @param span_period A `"Y"` scalar character. If `"Y"`, events that started prior
-#' to the `ref_start_date`and are ongoing or end after the `ref_start_date` are flagged as `"Y"`.
-#'  Optional; default is `NULL`.
-#'
-#' @details
-#' On-Treatment is calculated by determining whether the assessment date or
-#' start/stop dates fall between 2 dates. The following logic is used to
-#' assign on-treatment = `"Y"`:
-#'   1. `start_date` is missing and `ref_start_date`is non-missing
-#'   2. No timepoint filter is provided (`filter_pre_timepoint`) and both
-#'      `start_date` and `ref_start_date` are non-missing and `start_date` =
-#'      `ref_start_date`
-#'   3. A timepoint is provided (`filter_pre_timepoint`) and both `start_date`
-#'      and `ref_start_date` are non-missing and `start_date = ref_start_date`
-#'      and the filter provided in `filter_pre_timepoint` is not true.
-#'   4. `ref_end_date` is not provided and `ref_start_date < start_date`
-#'   5. `ref_end_date` is provided and `ref_start_date < start_date` <=
-#'      `ref_end_date + ref_end_window`.
-#'
-#' If the `end_date` is provided and the `end_date` < ref_start_date
-#' then the `ONTRTFL` is set to `NULL`.This would be applicable to cases where
-#' the `start_date` is missing and `ONTRTFL` has been assigned as `"Y"` above.
-#'
-#' If the `span_period` is specified as `"Y"`, this allows the user to assign `ONTRTFL` as `"Y"` to
-#' cases where the record started prior to the `ref_start_date` and
-#' was ongoing or ended after the `ref_start_date`.
+#'   on-treatment when `date` = `ref_start_date`. For example, if observations
+#'   where `VSTPT = PRE` should not be considered on-treatment when `date =
+#'   ref_start_date`, `filter_pre_timepoint` should be used to denote when the
+#'   on-treatment flag should be set to null. Optional; default is `NULL`.
+#'
+#' @param span_period A `"Y"` scalar character. If `"Y"`, events that started
+#'   prior to the `ref_start_date`and are ongoing or end after the
+#'   `ref_start_date` are flagged as `"Y"`. Optional; default is `NULL`.
+#'
+#' @details On-Treatment is calculated by determining whether the assessment
+#' date or start/stop dates fall between 2 dates. The following logic is used to
+#' assign on-treatment = `"Y"`: 1. `start_date` is missing and
+#' `ref_start_date`is non-missing 2. No timepoint filter is provided
+#' (`filter_pre_timepoint`) and both `start_date` and `ref_start_date` are
+#' non-missing and `start_date` = `ref_start_date` 3. A timepoint is provided
+#' (`filter_pre_timepoint`) and both `start_date` and `ref_start_date` are
+#' non-missing and `start_date = ref_start_date` and the filter provided in
+#' `filter_pre_timepoint` is not true. 4. `ref_end_date` is not provided and
+#' `ref_start_date < start_date` 5. `ref_end_date` is provided and
+#' `ref_start_date < start_date` <= `ref_end_date + ref_end_window`.
```

Comment:
This is no longer rendered properly.

Reply:
It seems that using `Code|Reflow Comment` to wrap the comments automatically somehow will wrap sentences even though an `enter` key is there. So after wrap the comments automatically, I manually added `enter` key between numbered sentences. Now the rendering looks fine to me.